tests/unittest_nodes.py

```diff
@@ -1583,6 +1583,14 @@ def func():
     """
     )
     assert node.doc == "Docstring"
+    if PY38_PLUS:
+        assert isinstance(node.doc_node, nodes.Const)
+        assert node.doc_node.lineno == 3
+        assert node.doc_node.col_offset == 4
+        assert node.doc_node.end_lineno == 3
+        assert node.doc_node.end_col_offset == 15
+    else:
+        assert not node.doc_node
```

Comment:
Let's do:
```suggestion
        assert node.doc_node is None
```

If `pylint` starts complaining let me know as then I think we should disable that message here (or in pylintrc). We want to know `node.doc_node is None` and not `node.doc_node == []` (which would pass this assertion).